mdtraj/core/trajectory.py

```diff
@@ -1669,6 +1669,76 @@ def remove_solvent(self, exclude=None, inplace=False):
 
         return self.atom_slice(atom_indices, inplace = inplace)
 
+    def smooth(self, width=11, order=3, atom_indices=None, inplace=False):
+        """
+        Smoothen a trajectory using a zero-delay Buttersworth filter.
+        This code is copied from the scipy cookbook, with sytlistic improvements.
+        http://www.scipy.org/Cookbook/FiltFilt
+        
+        Parameters
+        ----------
+        width : float
```

Comment:
Yeah that was left over from @rmcgibbo's code. He has this line that converts it into an odd integer:

`pad = int(np.ceil((width + 1)/2)*2 - 1)`

Philosophically, I think the user should know that `width` has to be an odd whole number so they have at least some idea of what's going on under the hood, but maybe that's asking for too much?